pkg/sql/parser/parse_test.go

```diff
@@ -760,6 +760,7 @@ var nonBareLabelKeywords = map[string]struct{}{
 	"INTERSECT": {},
 	"INTO":      {},
 	"LIMIT":     {},
+	"LOGGED":    {},
```

Comment:
instead of adding an exception here, `LOGGED` should just be added to `bare_label_keywords` in sql.y